docs/source/java/overview.rst

```diff
@@ -0,0 +1,218 @@
+
+
+Arrow Java Modules
+------------------
+
+Arrow Java are divided in these modules to offer in-memory columnar data structures:
+
+.. list-table:: Arrow Java Modules
+   :widths: 25 50 25
+   :header-rows: 1
+
+   * - Module
+     - Description
+     - Root Decision
+   * - arrow-format
+     - Generated Java files from the IPC Flatbuffer definitions.
+     - Native implementation
+   * - arrow-memory-core
+     - Core off-heap memory management libraries for Arrow ValueVectors.
+     - Native implementation
+   * - arrow-memory-unsafe
+     - Allocator and utils for allocating memory in Arrow based on sun.misc.Unsafe.
+     - Native implementation
+   * - arrow-memory-netty
+     - Netty allocator and utils for allocating memory in Arrow.
+     - Native implementation
+   * - arrow-vector
+     - An off-heap reference implementation for Arrow columnar data format.
+     - Native implementation
+   * - arrow-tools
+     - Java applications for working with Arrow ValueVectors.
+     - Native implementation
+   * - arrow-jdbc
+     - (Experimental) A library for converting JDBC data to Arrow data.
+     - Native implementation
+   * - arrow-plasma
+     - (Experimental) Java client for the Plasma object store.
+     - Native implementation
+   * - flight-core
+     - (Experimental) An RPC mechanism for transferring ValueVectors.
+     - Native implementation
+   * - flight-grpc
+     - (Experimental) Contains utility class to expose Flight gRPC service and client.
+     - Native implementation
+   * - flight-sql
+     - (Experimental) Contains utility classes to expose Flight SQL semantics for clients and servers over Arrow Flight.
+     - Native implementation
+   * - flight-integration-tests
+     - Integration tests for Flight RPC.
+     - Native implementation
+   * - arrow-performance
+     - JMH Performance benchmarks for other Arrow libraries.
+     - Native implementation
+   * - arrow-algorithm
+     - (Experimental) A collection of algorithms for working with ValueVectors.
+     - Native implementation
+   * - arrow-avro
+     - (Experimental) A library for converting Avro data to Arrow data.
+     - Native implementation
+   * - arrow-compression
+     - (Experimental) A library for working with the compression/decompression of Arrow data.
+     - Native implementation
+   * - arrow-c-data
+     - Java implementation of C Data Interface
+     - Bindings
+   * - arrow-orc
+     - (Experimental) A JNI wrapper for the C++ ORC reader implementation.
+     - Bindings
+   * - arrow-gandiva
+     - Java wrappers around the native Gandiva SQL expression compiler.
+     - Bindings
+   * - arrow-dataset
+     - Java implementation of Arrow Dataset API/Framework
+     - Bindings
+
+Arrow Java modules offer support to work data (1) in-memory, (2) at rest and (3) on-the-wire.
+
+For more detail about how to install this modules please review
+:doc:`Installing Java Modules <install>`.
```

Comment:
This is already in the sidebar, no need to repeat IMO

Reply:
Deleted